suite-native/transactions/src/components/TransactionDetail/NetworkTransactionDetailSummary.tsx

```diff
@@ -7,11 +7,28 @@ import {
     selectTransactionByTxidAndAccountKey,
 } from '@suite-common/wallet-core';
 import { AccountKey } from '@suite-common/wallet-types';
-import { ErrorMessage, VStack } from '@suite-native/atoms';
+import { Box, ErrorMessage, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 import { selectTransactionAddresses } from '../../selectors';
 import { TransactionDetailAddressesSection } from './TransactionDetailAddressesSection';
-import { TransactionDetailStatusSection } from './TransactionDetailStatusSection';
+
+const separatorStyle = prepareNativeStyle<{ isMultiAddress: boolean }>(
+    (utils, { isMultiAddress }) => ({
+        position: 'absolute',
+        left: 7.5,
+        top: 37,
+        width: 1,
+        height: isMultiAddress ? 90 : 34,
```

Comment:
Is the static height really needed? And why 90 for "multi address"? There can be more outputs...

Reply:
There are only two options defined by design:

A) Single input/output:
![Screenshot 2024-09-04 at 10 40 38](https://github.com/user-attachments/assets/48c33953-489f-41f2-971c-8c7cb9e68bdd)

B) Multiple inputs/outputs:
![Screenshot 2024-09-04 at 10 34 23](https://github.com/user-attachments/assets/e06c1ec5-8137-40b2-8ea1-02d97a7a8e1d)


For this reason I believe that no dynamic height checking is needed. I at least improved the naming and put the static values to `const`s.  59369acb4d

Reply:
I get it now